libobs/obs-cocoa.m

```diff
@@ -461,8 +461,11 @@ static bool code_to_str(int code, struct dstr *str)
 
 void obs_key_to_str(obs_key_t key, struct dstr *str)
 {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wvla"
     const UniCharCount max_length = 16;
```

Comment:
I wonder if in this case it wouldn't be simpler to just do `UniChar buffer[16]` given that the value is very much static.

Reply:
This is not resolved - it replaced a variable-length array with a magic number and preprocessor statements, all of which are superfluous. The number doesn't change, so it can and should be represented in code.

Reply:
No, it doesn't become a VLA since it's a macro and the magic number was already there before the macro statement.